src/WebJobs.Extensions.DurableTask/ContextImplementations/DurableOrchestrationContext.cs

```diff
@@ -4,15 +4,19 @@
 using System;
 using System.Collections;
 using System.Collections.Generic;
+using System.Diagnostics;
 using System.Linq;
 using System.Net;
 using System.Net.Http;
 using System.Reflection;
 using System.Runtime.ExceptionServices;
 using System.Threading;
 using System.Threading.Tasks;
+using Azure.Core;
 using DurableTask.Core;
 using DurableTask.Core.Exceptions;
+using DurableTask.Core.Tracing;
+using Microsoft.Azure.WebJobs.Extensions.DurableTask.Correlation;

```

Comment:
Can we remove `using Azure.Core` and `using Microsoft.Azure.WebJobs.Extensions.DurableTask.Correlation`?